CHANGES.md

```diff
@@ -1,3 +1,7 @@
+1.3.2 (2016-10-25)
+==================
+Bump dependency on matrix-org/matrix-appservice-node from `0.3.1` to `0.3.3`. See (https://github.com/matrix-org/matrix-appservice-node/blob/1ff56e9c11d8536f2ce7043279818bfa61b8fa91/CHANGELOG.md#v033matrix-appservice-node#)
```

Comment:
I would put a space between the brackets or use proper Markdown linking via `[url.com](some text)` else some browsers will include the `)` in the URL.